File: task-03/Sub-Task 2/Ruby/code.rb
file = File.open("/Users/a.t.yaswanthreddy/VS Code/Club/Task 1 Sub Task 2/Ruby/input.txt", "r+")

output = File.open("/Users/a.t.yaswanthreddy/VS Code/Club/Task 1 Sub Task 2/Ruby/output.txt", "r+")
output.puts(file.read)
    
file.close()
output.close()

File: task-03/Sub-Task 2/JavaScript/code.js
const { isUtf8 } = require('buffer');
const fs = require('fs');

fs.readFile('/Users/a.t.yaswanthreddy/VS Code/Club/Task 1 Sub Task 2/Ruby/input.txt', 'utf8', (err, data) => {
    if(err){
        console.error(err);
        return;
    }

    fs.writeFile('output.txt', data, err => {
    if (err){
        console.err;
        return;
        }
    });
});
